testWork2/task2/main.c

```diff
@@ -0,0 +1,141 @@
+#include <stdio.h>
+#include <string.h>
+#include "stack.h"
+#include <stdlib.h>
```

Comment:
```suggestion
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include "stack.h"
```
Стоит упорядочивать подключаемые модули. Сначала стандартные, потом кастомные, но не твои, в конце твои.